### PR TITLE
V3.0.0 rc doc

### PR DESCRIPTION
**Pull request by scott-w**
I've written up the `CollectionView` migration examples to migrate users from `CompositeView`. It currently just fits into the existing docs but I think that'll be sufficient for anyone using the v3 RC.

If you guys have anything you'd like to add, or any suggestions, please let me know and I'll get them in as soon as time allows.

### CODE REVIEW

File: docs/marionette.collectionview.md [hunk 1]
```diff
@@ -457,6 +465,295 @@ myView.destroyChildren({checkEmpty: false}); // will not show emptyView
...
+  tagName: 'tr',
+  template: '#row-template'
+});
+
+var TableBody = Marionette.CollectionView.extend({
+  tagName: 'tbody',
+  childView: RowView
+});
+
+var TableView = Marionette.View.extend({
```
Comment (urosgruber) [inline docs/marionette.collectionview.md]: Isn't this LayoutView?

Comment (paulfalgout) [inline docs/marionette.collectionview.md]: Marionette v3 will only have View CollectionView and the deprecated CompositeView.  So you'll be able to add regions to any view.

Comment (scott-w) [inline docs/marionette.collectionview.md]: I'll have a look at the `View` documentation and clearly label it there.

I think we're going to need to keep the existing documentation with links to the updated versions of everything at least for the initial versions so ease the transition.

**Comment by scott-w** [comment]
This is ready to merge once the test is passing

**Comment by paulfalgout** [comment]
The failing test is an eslint dependency issue.  We can come back to this if there's a problem.

**Pull request merged**